test/teiserver_web/tachyon/matchmaking_test.exs

```diff
@@ -3,52 +3,111 @@ defmodule Teiserver.Tachyon.MatchmakingTest do
   alias Teiserver.Support.Tachyon
   alias Teiserver.OAuthFixtures
   alias Teiserver.Player
+  alias Teiserver.AssetFixtures
+  alias Teiserver.Asset
 
-  describe "list" do
-    setup {Tachyon, :setup_client}
+  def altair_attr() do
+    altair_attr("8v8")
+  end
 
-    test "works", %{client: client} do
-      resp = Tachyon.list_queues!(client)
+  def altair_attr(id),
+    do: %{
+      spring_name: "Altair Crossing Remake 1.2.3",
+      display_name: "Altair Crossing",
+      thumbnail_url: "https://www.beyondallreason.info/map/altair-crossing",
+      matchmaking_queues: ["1v1", "2v2", "3v3", id]
```

Comment:
why do you need some defaults for the `matchmaking_queues`?

Reply:
I don't, but thought there was no harm in putting some as an example.

Reply:
Removed to not cause confusion